GTG/plugins/send_email/sendEmail.py

```diff
@@ -55,18 +72,58 @@ def deactivate(self, plugin_api):
         Desactivates the plugin.
         """
         # everything should be removed, in case a task is currently opened
-        try:
-            self.plugin_api.remove_toolbar_item(self.tb_Taskbutton)
-        except:
-            pass
+
+        if self.tb_Taskbutton:
+            self.taskview_plugin_api.remove_toolbar_item(self.tb_Taskbutton)
+            self.tb_Taskbutton = None
+            self.browser_plugin_api.remove_toolbar_item(self.tb_Managerbutton)
+            self.tb_Managerbutton = None
+
+        if self.tb_Managerbutton:
+            self.browser_plugin_api.remove_toolbar_item(self.tb_Managerbutton)
+            self.tb_Managerbutton = None
+
+
+
+    def selection_changed(self, selection):
+        """
+        To select multiple tasks in manager
+        deactivates button if no task selection_changed
+        """
+        if selection.count_selected_rows() > 0:
+            self.tb_Managerbutton.set_sensitive(True)
+        else:
+            self.tb_Managerbutton.set_sensitive(False)
+
+    def get_new_button(self):
+        """ Initialize a new button """
+        button = Gtk.ToolButton()
+        button.set_sensitive(False)
+        button.set_icon_name('mail-send')
+        button.set_is_important(True)
```

Comment:
is_important is needed to display image with label in button, otherwise only image will be displayed in button.